PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -126,31 +126,31 @@
 )
 process.patJetsAK4PF.addTagInfos = True
 
-# uncomment the following lines to add ak8PFJetsCHS with new b-tags to your PAT output
+# uncomment the following lines to add ak8PFJetsPuppi with new b-tags to your PAT output
```

Comment:
it looks like the comment is obsolete now
or perhaps it should say the reverse "comment out .. to remove "?

Reply:
As you recommend below I will just leave the puppi out of the tests for the time being until things are integrated.